src/app/FailSafeContext.h

```diff
@@ -66,7 +66,7 @@ class FailSafeContext
 
     bool IsFailSafeArmed(FabricIndex accessingFabricIndex) const
     {
-        return mFailSafeArmed && MatchesFabricIndex(accessingFabricIndex);
+        return IsFailSafeArmed() && MatchesFabricIndex(accessingFabricIndex);
```

Comment:
As below, is this a recursive call?

Reply:
It's not; there are two different overloads of IsFailSafeArmed.